### PR TITLE
Fusion v2 updates: AWS Batch Tower Forge CE instructions and defaults

### PR DESCRIPTION
**Pull request by gwright99**
@llewellyn-sl - **Dont approve this as is.**

### CODE REVIEW

**Comment by gwright99** [comment]
@llewellyn-sl - I had a bunch of content written that I then ripped out since I realized the https://www.nextflow.io/docs/latest/ docs had a bunch of the material covered already. 

With that said, there are still gaps / discrepancies / missing sections. I've left 8 points and related commentary in a section called "@Llewellyn - Thoughts and feedback".  I can discuss tomorrow at your convenience.

File: docs/supported_software/fusion/fusion.md [hunk 1]
```diff
@@ -1,21 +1,210 @@
...
 
-## Fusion requirements
+
+# @Llewellyn - Thougtht and feedback 
+1. I think we want to be more prescriptive to our commercial customers. Existing documentation (i.e. blog post and original content on this page) is wishy-washy re: recommended storage: blog shows lousy EBS-based Fusion run alongside NVME. This content originally had NVME usage as optional. [Nextflow fusion](https://www.nextflow.io/docs/latest/fusion.html#nvme-storage) page recommends NVME for max performance but that's in the 3rd paragraph of the bottom section.
+
+2. Networking is assumed to be reliable and continuous. This is not always true in a cloud environment. We do not talk about it but I'm convinced our paying clients will encounter this problem. TBD whether that goes here or should be added to the Nextflow Fusion docs (I've kept it here for now).
```
Comment (llewellyn-sl) [inline docs/supported_software/fusion/fusion.md:13]: At least a note, and a link out to NF if needed. Call out the network and storage need to get Fusion to deliver on its promises. 

https://instances.vantage.sh/ 

Prescriptive suggestions are needed for clients actually to get the Fusion advantage. Fusion is for people willing to go big on network, disk size, etc.

File: docs/supported_software/fusion/fusion.md [hunk 2]
```diff
@@ -1,21 +1,210 @@
...
+
+3. Batch Forge offers a **Boot disk size** option which I can't find in the docs. We seem to force 50GB at least, and if larger numbers are specified it makes initial AWS EC2 initializational glacial. This will impact perceived performance (for storage that I'm under the impression isn't required). There should probably be a warning.
+
+4. Tower Forge has undocumented features when creating an AWS Batch NVME environment:
```
Comment (llewellyn-sl) [inline docs/supported_software/fusion/fusion.md:17]: Recommend larger instances of the (wider) list of family types (networking limitations pointed to above).

Manual entry of instance types during CE creation (AWS for now) 

Collapsible code block with recommended instances for client to paste into instance types.

File: docs/supported_software/fusion/fusion.md [hunk 3]
```diff
@@ -1,21 +1,210 @@
...
+        ```
+
+#### 3. Configure the Nextflow Pipeline
+
+Use Nextflow version `22.10.0` or later. The latest version of Nextflow is used in Tower by default, but a particular version can be specified using `NXF_VER` in the Nextflow config file field (**Advanced options -> Nextflow config file** under Pipeline settings on the launch page). 
+
+=== "Tower Launch"
+
+    1. Add the following configuration values to your *nextflow.config*:
+        ```
```
Comment (bentsherman) [inline docs/supported_software/fusion/fusion.md:177]: @llewellyn-sl Nextflow config snippets can be annotated as `groovy` for basic syntax highlighting

Comment (robsyme) [inline docs/supported_software/fusion/fusion.md:177]:
```suggestion
        ```groovy
```

File: docs/supported_software/fusion/fusion.md [hunk 4]
```diff
@@ -1,21 +1,210 @@
...
+        aws.batch.volumes = '/scratch/fusion:/tmp' 
+        process.scratch = false
+        ```
+
+=== "Nextflow CLI Launch"
+
+    1. Add the following configuration values to your *nextflow.config*:
+        ```
```
Comment (robsyme) [inline docs/supported_software/fusion/fusion.md:185]:
```suggestion
        ```groovy
```

File: docs/supported_software/fusion/fusion.md (hunk 2 above)
Comment (llewellyn-sl) [reply docs/supported_software/fusion/fusion.md:17]: https://support.seqera.io/a/tickets/3104

File: docs/supported_software/fusion/fusion.md [hunk 5]
```diff
@@ -2,20 +2,257 @@
 description: 'Fusion file system'
 ---
 
-Tower 22.4 adds official support for the Fusion file system. Fusion is a lightweight client that enables containerized tasks to access data in Amazon S3 (and other object stores in future) using POSIX file access semantics. Depending on your data handling requirements, Fusion 2.0 improves pipeline throughput and/or reduces cloud computing costs. See [here](https://seqera.io/fusion/) for more information on Fusion's features. 
+# Fusion
```
Comment (llewellyn-sl) [inline docs/supported_software/fusion/fusion.md:5]:
```suggestion
## Fusion file system
```

File: docs/supported_software/fusion/fusion.md [hunk 6]
```diff
@@ -1,27 +1,280 @@
...
+fusion {
+  enabled = true
+    }
+```
 
-- Enable the [Wave containers service](https://www.nextflow.io/docs/latest/wave.html#wave-page) during [AWS Batch](/docs/compute-envs/aws-batch.md) compute environment creation.
+### Fusion performance and cost considerations
 
-- Select **Enable Fusion v2** during compute environment creation.
+Fusion improves pipeline throughput for containerized tasks by simplifying direct access to cloud data storage. Data throughput depends on network throughput, so Fusion performance is heavily dependent on networking capabilities. Network throttling on smaller AWS instances, for example, limits the impact Fusion will have on your pipeline execution times.
```
Comment (jordeu) [inline docs/supported_software/fusion/fusion.md:25]: I think this:

"Data throughput depends on network throughput, so Fusion performance is heavily dependent on networking capabilities. Network throttling on smaller AWS instances, for example, limits the impact Fusion will have on your pipeline execution times." 

is ambiguous and I do not fully agree. In fact Fusion is decoupling the write of file to disk from their upload so, of course good network is better, but this is not a Fusion thing.

File: docs/supported_software/fusion/fusion.md [hunk 7]
```diff
@@ -1,27 +1,280 @@
...
 
-- Enable the [Wave containers service](https://www.nextflow.io/docs/latest/wave.html#wave-page) during [AWS Batch](/docs/compute-envs/aws-batch.md) compute environment creation.
+### Fusion performance and cost considerations
 
-- Select **Enable Fusion v2** during compute environment creation.
+Fusion improves pipeline throughput for containerized tasks by simplifying direct access to cloud data storage. Data throughput depends on network throughput, so Fusion performance is heavily dependent on networking capabilities. Network throttling on smaller AWS instances, for example, limits the impact Fusion will have on your pipeline execution times. 
 
-- (Optional) Select **Enable fast instance storage** to make use of NVMe instance storage to further increase performance.
+!!! note "AWS instance networking limitations"
+    Whether using Fusion with NVMe or EBS-only AWS instances, instance networking performance has a considerable impact on Fusion v2's performance. We recommend 8xlarge or greater instances for large production pipelines to deliver increased workflow performance.
```
Comment (jordeu) [inline docs/supported_software/fusion/fusion.md:28]: here happens the same, of course this is better, but not only for Fusion. I mean you do not need to do this to use Fusion.

File: docs/supported_software/fusion/fusion.md [hunk 8]
```diff
@@ -1,27 +1,280 @@
...
+    - If left unspecified, Tower will select the following NVMe-based instance type families: `'c5ad', 'c5d', 'c6id', 'i3', 'i4i', 'm5ad', 'm5d', 'm6id', 'r5ad', 'r5d', 'r6id'`
+    - To specify an NVMe instance family type, select from the following: 
+        - Intel: `'c5ad', 'c5d', 'c6id', 'dl1', 'f1', 'g4ad', 'g4dn', 'g5', 'i3', 'i3en', 'i4i', 'm5ad', 'm5d', 'm5dn', 'm6id', 'p3dn', 'p4d', 'p4de', 'r5ad', 'r5d', 'r5dn', 'r6id', 'x2idn', 'x2iedn', 'z1d'`
+        - ARM: `'c6gd', 'm6gd', 'r6gd', 'x2gd', 'im4gn', 'is4gen'`
+
+    !!! note "Optimal instance type families will not work with fast instance storage"
+        When enabling fast instance storage, do not select the `optimal` instance type families (c4, m4, r4) for your compute environment as these are not NVMe-based instances. Specify the NVMe instance types listed above.
+
+!!! note
+    We recommend selecting 8xlarge or above for large and long-lived production pipelines. Dedicated networking ensures a guaranteed network speed service level compared with "burstable" instances. See [Instance network bandwidth](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-network-bandwidth.html) for more information.
```
Comment (jordeu) [inline docs/supported_software/fusion/fusion.md:50]: I think that to say this only once here is enough.

File: docs/supported_software/fusion/fusion.md (hunk 7 above)
Comment (llewellyn-sl) [inline docs/supported_software/fusion/fusion.md:28]:
```suggestion
Dedicated networking and fast I/O influence pipeline performance and are important to consider when selecting compute instances.
```

File: docs/supported_software/fusion/fusion.md [hunk 9]
```diff
@@ -1,27 +1,280 @@
...
+    - EBS autoscaling is disabled
+    - EBS boot disk size is increased to 100GB
+    - EBS boot disk type is changed to GP3
+    - EBS boot disk throughput is increased to 325MB/s
+
+6. Use an S3 bucket as the pipeline work directory. 
+
+#### Fusion in manual compute environments 
+
+Tower supports Fusion v2 for both Tower Forge and manual compute environments on AWS Batch and Google Cloud Batch. Detailed instructions for Fusion configuration in manual compute environments will be available soon. In the meantime, we highly recommend using Fusion in Tower Forge compute environments with the default settings described above.
```
Comment (llewellyn-sl) [inline docs/supported_software/fusion/fusion.md:74]:
```suggestion
Tower supports Fusion v2 for both Batch Forge and manual compute environments on AWS Batch, and for Google Cloud Batch compute environments. Detailed instructions for configuration in manual AWS and Google Cloud Batch compute environments will be available soon. In the meantime, we highly recommend using Fusion v2 in Batch Forge compute environments with the default settings described above. 
```

File: docs/supported_software/fusion/fusion.md [hunk 10]
```diff
@@ -1,27 +1,280 @@
...
+```
 
-- Enable the [Wave containers service](https://www.nextflow.io/docs/latest/wave.html#wave-page) during [AWS Batch](/docs/compute-envs/aws-batch.md) compute environment creation.
+### Fusion performance and cost considerations
 
-- Select **Enable Fusion v2** during compute environment creation.
+Fusion improves pipeline throughput for containerized tasks by simplifying direct access to cloud data storage.  
 
-- (Optional) Select **Enable fast instance storage** to make use of NVMe instance storage to further increase performance.
+!!! note "AWS instance networking limitations"
```
Comment (llewellyn-sl) [inline docs/supported_software/fusion/fusion.md:27]:
```suggestion
```

File: docs/supported_software/fusion/fusion.md (hunk 6 above)
Comment (llewellyn-sl) [reply docs/supported_software/fusion/fusion.md]: As discussed, statements about networking and performance have been reworded.

**Review by llewellyn-sl — APPROVED** [review]

**Comment by llewellyn-sl** [comment]
@jordeu @robsyme although this PR has been reworked and approved by Graham and myself, please give it a last look before we merge and publish.

**Pull request merged**